### ISSUE TITLE
Feature Request: Make a lighting engine mode with a fancy mario model and graphics!

### ISSUE THREAD

**Issue by BeezBumba** [rovertronic/Mario-Builder-64#43]
Make a lighting engine mode with a fancy mario model and graphics!

**Comment by Mariosonicman91** [comment]
This'd be a feature only if the Rom Hack got a Decomp Port in the future, which is far from now. An N64 emulator EMULATES AN N64. They sure as heck cannot replicate shaders. I think. It tricks the software into thinking it's an N64, but it's not really an N64, huh?

I looked it up. I did use COPILOT (Which has surprisingly helped me a lot) and it says a custom build can implement shaders. So apparently, I was wrong. It is peculiar though, since the game chugs from too many objects or tiles. It could be entirely wrong.

TLDR, pretty sure we have to wait for a "PC" Port.

EDIT: Fixed me using TTYL instead of TLDR, haven't used either in a long time.
